techniques/silverlight/SL12.html

```diff
@@ -21,18 +21,18 @@
                <a href="https://msdn.microsoft.com/en-us/library/system.windows.controls.mediaelement.stop%28VS.95%29.aspx">Stop</a> 
             </li>
       </ul>
-      <p>Note that by default, a <code class="obj">MediaElement</code> will start playing
+      <p>Note that by default, a <code>MediaElement</code> will start playing
     				its media as soon as the UI loads completely AND the media source file
     				is downloaded (or a certain buffer size is reached, in the case of
-    				streaming media). Use the <code class="prop">AutoPlay</code> property to change
+    				streaming media). Use the <code>AutoPlay</code> property to change
     				this default. </p>
    </section><section id="examples"><h2>Examples</h2>
       <section class="example">
          <h3>Providing MediaElement controls in the UI</h3>
          
             <p>This example has a UI definition in XAML and interaction logic in
     					C#. </p>
-            <pre xml:space="preserve">&lt;UserControl x:Class="MediaElementControls.MainPage"
+            <pre>&lt;UserControl x:Class="MediaElementControls.MainPage"
```

Comment:
```suggestion
            <pre><code>&lt;UserControl x:Class="MediaElementControls.MainPage"
```
and add matching `</code>`